README.md

```diff
@@ -15,8 +15,9 @@ Note, branch `asf-site` don't accept pull request, all `asf-site` branch HTMLs s
 Usage:
 
 1. Git pull branch `asf-site`
-2. Run `npm run build`
-3. Push files and send pull request if necessary.
+2. Run `npm run clean`
```

Comment:
How about add **`npm run dev` in your local env to check if it works fine.And then,**